### PR TITLE
add WithPlatform in bundle unpack

### PR DESCRIPTION
**Pull request by nianyush**
Signed-off-by: Nianyu Shen <xiaoyu9964@gmail.com>

### CODE REVIEW

File: bundles/bundles.go [hunk 1]
```diff
@@ -204,10 +213,16 @@ func (e OCIImageExtractor) Install(config *BundleConfig) error {
 	if err != nil {
 		return err
 	}
+
+	platform := config.Platform
+	if len(platform) == 0 {
+		platform = utils.GetCurrentPlatform()
+	}
```
Comment (mauromorales) [inline bundles/bundles.go:220]: Sounds to me like we never want to use `config.Platform` directly. Could you instead add a function, e.g. `BundleConfig.Platform()` that will do this operation?

Comment (mauromorales) [reply bundles/bundles.go:220]: can we also have tests for the added code?

Comment (nianyush) [reply bundles/bundles.go:220]: sure i will make changes

File: bundles/bundles.go [hunk 2]
```diff
@@ -233,10 +248,16 @@ func (e OCIImageRunner) Install(config *BundleConfig) error {
 	if err != nil {
 		return err
```
Comment (nianyush) [inline bundles/bundles.go:249]: found out that GetImage is already handling empty platform case. So we dont need to handle it here. https://github.com/nianyush/kairos-sdk/blob/7d11ed67c526ce2477bf2103a92fb405a90ff36a/utils/image.go#L80

File: bundles/bundles.go (hunk 1 above)
Comment (nianyush) [reply bundles/bundles.go]: @mauromorales can you review again?

**Review by mauromorales — APPROVED** [review]

**Pull request merged**